app/src/test/java/org/simple/clinic/setup/SetupActivityEffectHandlerTest.kt

```diff
@@ -80,4 +85,20 @@ class SetupActivityEffectHandlerTest {
     testCase.assertOutgoingEvents(DatabaseInitialized)
     verifyZeroInteractions(uiActions)
   }
+
+  @Test
+  fun `the logged in user must be fetched when the fetch user details effect is received`() {
+    // given
+    whenever(onboardingCompletePreference.get()) doReturn false
+    val user = PatientMocker.loggedInUser(uuid = UUID.fromString("426d2eb9-ebf7-4a62-b157-1de221c7c3d0"))
+    whenever(userDao.userImmediate()).doReturn(user)
+
+    // when
+    testCase.dispatch(FetchUserDetails)
+
+    // then
+    val expected = UserDetailsFetched(hasUserCompletedOnboarding = false, loggedInUser = Just(user))
+    testCase.assertOutgoingEvents(expected)
```

Comment:
`expected`? Can we use a more descriptive name here?

Reply:
This test has been removed after unifying the different conditions for fetching user details.